GHzDACs/Cleanup/dac.py

```diff
@@ -205,9 +275,11 @@ def pktWriteSram(cls, derp, data):
                If less than a full derp is written, the rest of the derp is
                populated with zeros.
         """
+        # TODO: I think this function is unneccessary. I think data.tostring()
+        # should work if data.dtype == '<u4' (and is of length 256).
```

Comment:
If not for the leading 2-byte write address, this would be correct. As it stands however, I think this comment should be removed.